src/client/action/list_databases.rs

```diff
@@ -90,8 +90,8 @@ impl<'a, M> ListDatabases<'a, M> {
     );
 
     /// Runs the query using the provided session.
-    pub fn session(mut self, value: &'a mut ClientSession) -> Self {
-        self.session = Some(value);
+    pub fn session(mut self, value: impl Into<&'a mut ClientSession>) -> Self {
```

Comment:
This needed to become `Into` so it could accept both sync and async `ClientSession`s.